internal/model/board.go

```diff
@@ -319,17 +352,56 @@ func NewBoard(size int) (*Board, error) {
 	})
 
 	board := &Board{
-		Tiles: make([][]*BoardTile, size),
-		Players: []*Player{
-			{
-				Color:   ColorBlue,
-				Line:    0,
-				Row:     0,
-				Targets: treasures,
-				Score:   0,
-			},
-		},
-		State: GameStatePlaceTile,
+		Tiles:                make([][]*BoardTile, size),
+		Players:              make([]*Player, 0, playerCount),
+		RemainingPlayers:     make([]int, 0, playerCount),
+		RemainingPlayerIndex: 0,
+		State:                GameStatePlaceTile,
+	}
+
+	board.Players = append(board.Players, &Player{
```

Comment:
couldn't you represent all that logic into one data object, and loop over it? In JS, I'd go

```js
const playerPositions = [
  { color: 'blue', x: 1, y: 1 },
  { color: 'green', x: 1, y: -1 },
  // etc
]
```

This makes a data structure emerge and removes code duplication.